.github/workflows/pw_aws_ci.yaml

```diff
@@ -114,7 +114,7 @@ jobs:
     - name: Build components
       run: |
         cd ${{ env.TEST_DIR }}/HOMEgfs/sorc
-        ./build_all.sh -j 8
+        ./build_all.sh -w -j 16 
```

Comment:
This will build the GEFS variant of the model (as is required for the GEFS test).  Doing so, will no longer test GFS tests.  
What/How do we plan to test GFS and GEFS simulateously?